packages/site/src/components/Nav.js

```diff
@@ -62,7 +64,7 @@ export default function Nav({ className, demos }: Props) {
   const demoLinks = Object.keys(demos).map(slug => {
     const demo = demos[slug];
     return (
-      <ListItem key={slug}>
+      <ListItem key={slug} indent={demo.subcomponent}>
```

Comment:
Consider renaming prop `indent` to `subcomponent` of `isSubcomponent`.  This way, you are not "limited" to indentation styles if the design changes later.